docs/reference/mapping/types/nested.asciidoc

```diff
@@ -71,10 +70,15 @@ GET my_index/_search
 ==== Using `nested` fields for arrays of objects
 
 If you need to index arrays of objects and to maintain the independence of
-each object in the array, you should use the `nested` datatype instead of the
-<<object,`object`>> datatype.  Internally, nested objects index each object in
+each object in the array, use the `nested` datatype instead of the
+<<object,`object`>> datatype.
+
+TIP: If you consider creating `nested` objects with two `key` and `value` keyword fields, consider using the <<flattened>> datatype instead.
```

Comment:
Maybe we could move this tip to the end of the section? I think it comes right in the middle of an important explanation and breaks the continuity.

Reply:
I'm considering a new section for **Interacted with nested documents** that contains the ways users can interact with these documents, this new tip, and the Important note that exists. For example:
![image](https://user-images.githubusercontent.com/25848033/82090713-0df2e500-96c4-11ea-977a-7b35d3454210.png)